src/praisonai-agents/tests/unit/context/test_conversation_simple.py

```
@@ -0,0 +1,53 @@
+    
+    try:
+        result, context = compactor.compact_conversation(
+            messages=messages,
+            target_tokens=200,
+            preserve_recent=2
+        )
+        print(f"✓ Compaction successful: {len(messages)} → {len(result)} messages")
+        print(f"  Context topic: {context.main_topic}")
+        print("✓ Conversation compaction test passed!")
+        return True
+    except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)